scripts/linking.py

```diff
@@ -71,19 +71,24 @@ class CandidateGenerator:
         The vectorizer used to encode mentions.
     ann_concept_id_list: List[str]
```

Comment:
nit: change parameter name (and description if needed)

Reply:
Done, thanks, FYI having correct docstrings is not a nit in my book